beginner_source/profiler_tutorial.py

```diff
@@ -0,0 +1,205 @@
+"""
+Profiling your PyTorch Module
+------------
+
+PyTorch includes a profiler API that is useful to identify the time and
+memory costs of various PyTorch operations in your code. It records the
+events of functions executed in C++, and exposes them to Python.
```

Comment:
it's not only C++, and we also have C++ version of profiler api, so let's remove this sentence.